### ISSUE TITLE
[Bug]: DSWx-S1 ISO Metadata Discrepencies with Product Spec

### ISSUE THREAD

**Issue by collinss-jpl** [nasa/opera-sds-pge#449]
### Checked for duplicates

Yes - I've already checked

### Describe the bug

As part of V&V testing for the DSWx-S1 PGE, the following discrepancies were reported between the ISO.xml and the metadata as described within the Product Spec Document (PSD):

1. ISO.xml specified an input of NOAA GSHHS for shoreline shape file, but nothing mentioned in the PSD
2. Could not find PROCESSING_INFORMATION_WORKFLOW in iso.xml 
3. PROCESSING_INFORMATION_THRESHOLD_TILE_AVERAGE in PSD is ProcessingInformationTileAverage in iso.xml (no Threshold) 
4. PROCESSING_INFORMATION_FILTER_WINDOW_SIZE in PSD is ProcessingInformationWindowSize (no Filter)

### What did you expect?

For finding 1, any references to the shoreline shapefile ancillary within the ISO.xml template for DSWx-S1 should be removed.

For finding 2, the metadata within a sample DSWx-S1 product needs to be examined to see if it is missing from there, or just from the iso.xml. If it is missing from the product, then ADT should update the Product Spec Document to remove PROCESSING_INFORMATION_WORKFLOW

For findings 3 and 4, the names used in the ISO.xml should be updated to match the names used in the PSD

### Reproducible steps

```bash
1.
2.
3.
...
```

### Environment

```bash
- Version of this software [e.g. vX.Y.Z]
- Operating System: [e.g. MacOSX with Docker Desktop vX.Y]
...
```

**Comment by collinss-jpl** [comment]
Latest product spec can be downloaded from Artifactory-FN under the path `general/gov/nasa/jpl/opera/adt/dswx_s1/r4.2/calval/documents/OPERA_DSWx_S1_ProductSpec_Calval_point_2_R4.2.docx`

The section of the ISO template with the product metadata starts here: https://github.com/nasa/opera-sds-pge/blob/977a059baaeb3284bd4832d1fd8c7dc6fde406d8/src/opera/pge/dswx_s1/templates/OPERA_ISO_metadata_L3_DSWx_S1_template.xml.jinja2#L576

**Comment by collinss-jpl** [comment]
After comparing DSWx-S1 product metadata with the product spec and the ISO metadata, the following has been determined:

- `PROCESSING_INFORMATION_WORKFLOW` is not present within the DSWx-S1 metadata. The CalVal product spec states that this field should be set to "DSWx-S1" as the workflow. Since this is probably redundant info, it should be removed from the product spec (will need to verify when documentation for Final delivery is provided)

All other changes listed in the ticket description will be addressed for the v3.0.0-rc.3.0 delivery of the DSWx-S1 PGE.
